python/cudf/cudf/core/column/column.py

```diff
@@ -102,6 +102,11 @@ def mask_array_view(self):
     def __len__(self):
         return self.size
 
+    def as_frame(self, name=None):
+        from cudf.core.frame import Frame
+
+        return Frame({name: self.copy(deep=False)})
```

Comment:
Is this a stopgap solution to prevent further refactoring of moving compute APIs from Column --> Frame?